dev-db/firebird/firebird-2.5.3.26780.0-r2.ebuild

```diff
@@ -238,17 +238,16 @@ pkg_config() {
 	# /etc/security2.fdb
 	if [ -f "${ROOT}/etc/firebird/security.gdb" ] ; then
 		# if we have scurity2.fdb already, back it 1st
-		if [ -f "${ROOT}/etc/firebird/security2.fdb" ] ; then
-			cp "${ROOT}/etc/firebird/security2.fdb" "${ROOT}/etc/firebird/security2.fdb.old"
-		fi
-		gbak -B "${ROOT}/etc/firebird/security.gdb" "${ROOT}/etc/firebird/security.gbk"
-		gbak -R "${ROOT}/etc/firebird/security.gbk" "${ROOT}/etc/firebird/security2.fdb"
-		mv "${ROOT}/etc/firebird/security.gdb" "${ROOT}/etc/firebird/security.gdb.old"
-		rm "${ROOT}/etc/firebird/security.gbk"
+		[[ -f "${ROOT}/etc/firebird/security2.fdb" ]] &&
+			cp "${ROOT}/etc/firebird/security2.fdb" "${ROOT}/etc/firebird/security2.fdb.old" || die
```

Comment:
This is going to die if "${ROOT}/etc/firebird/security2.fdb" doesn't exist. That's why you should use `if ...; then ... || die; fi`